CONTRIBUTING.md

```diff
@@ -134,7 +134,7 @@ flutter_tools' tests run inside the Dart command line VM rather than in the
 flutter shell. To run the test:
 
 * `cd packages/flutter_tools`
-* `dart --checked test/all.dart`
+* `pub run test -j1 -rexpanded`
```

Comment:
`-rexpanded` shouldn't matter much for local use since it just sets the reporter -- that said I'm entirely fine either way :)